lib/Login.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:onehabit/Dashboard.dart';
+import 'package:onehabit/themes/color.dart';
+
+class Login extends StatefulWidget {
+  const Login({Key? key}) : super(key: key);
+
+  @override
+  State<Login> createState() => _LoginState();
+}
+
+class _LoginState extends State<Login> {
+  var styleInput = const TextStyle(color: Colors.white);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Center(
+        child:
+            Column(mainAxisAlignment: MainAxisAlignment.spaceEvenly, children: [
+          SizedBox(width: 250),
+          Icon(Icons.person, color: Purple, size: 80),
+          const Text(
+            'Login',
+            style: TextStyle(fontFamily: 'outfit', fontSize: 55),
+          ),
+          SizedBox(
+            width: 250,
+            child: TextField(
+              decoration: new InputDecoration(
+                focusedBorder: OutlineInputBorder(
+                  borderSide: BorderSide(color: Purple, width: 4.0),
+                ),
+                enabledBorder: OutlineInputBorder(
+                  borderSide: BorderSide(color: Purple, width: 2.0),
+                ),
+                hintText: 'Enter your email',
+                prefixIcon: Icon(
+                  Icons.email,
+                  color: Purple,
+                ),
+              ),
+            ),
+          ),
+          SizedBox(
+            width: 250,
+            child: TextField(
+              decoration: new InputDecoration(
+                focusedBorder: OutlineInputBorder(
+                  borderSide: BorderSide(color: Purple, width: 4.0),
+                ),
+                enabledBorder: OutlineInputBorder(
+                  borderSide: BorderSide(color: Purple, width: 2.0),
+                ),
+                hintText: 'Enter your password',
+                prefixIcon: Icon(
+                  Icons.lock,
+                  color: Purple,
+                ),
+              ),
+            ),
+          ),
+          TextButton(
+            onPressed: () {
+              Navigator.push(
```

Comment:
Instead of calling a function within the widget. Better practice would be creating a function with a name, and calling that function. As a result, your code will become more readable. 
```suggestion
              navigateToDashboard
```